src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +102,20 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates.stream()
+                .filter(new CandidateValidator())
+                .map(Candidate::getName)
+                .sorted()
+                .collect(Collectors.toList());
+    }
+
+    private Predicate<Person> predicateForWorkablePeople(int fromAge, int femaleToAge,
```

Comment:
Method name must be verbs
```suggestion
    private Predicate<Person> getPredicateForWorkablePeople(int fromAge, int femaleToAge,
```